app/views/replies/_single.haml

```diff
@@ -32,13 +33,20 @@
       - if reply.id.present?
         = link_to post_or_reply_link(reply), rel: 'alternate'.freeze do
           = image_tag "icons/link.png".freeze, title: 'Permalink'.freeze, alt: 'Permalink'.freeze
-      - if reply.id.present? && (reply.editable_by?(current_user) || (reply.is_a?(Post) && reply.metadata_editable_by?(current_user)))
-        = link_to path_for(reply, 'edit_%s') do
-          = image_tag "icons/pencil.png".freeze, title: 'Edit'.freeze, alt: 'Edit'.freeze
-      - if reply.id.present? && reply.deletable_by?(current_user)
-        = link_to path_for(reply, '%s'), method: :delete, data: { confirm: "Are you sure you wish to delete this #{reply.class.to_s.downcase}?" } do
-          = image_tag "icons/cross.png".freeze, title: 'Delete'.freeze, alt: 'Delete'.freeze
-      - if reply.id.present? && reply.is_a?(Reply) && logged_in?
+      - if reply.id.present?
+        - if reply.order == 0 && reply.post.metadata_editable_by?(current_user)
+          = link_to path_for(reply.post, 'edit_%s') do
+            = image_tag "icons/pencil.png".freeze, title: 'Edit'.freeze, alt: 'Edit'.freeze
+        - elsif reply.editable_by?(current_user)
```

Comment:
I don't love the duplication of this "Edit" button, can we re-combine the condition? Either the reply is editable by the user, or the "reply" is the `post.written` and the post is editable by the user.

Or, I guess, we could segue this into fixing #218 sometime, using a different pencil? Maybe leave a comment in the codebase about why it's split up?

Reply:
…Oh, I see why it's separate – different paths. That's fair. Never mind.